kahuna/public/js/components/gu-date-range/gu-date-range.js

```diff
@@ -105,14 +108,11 @@ guDateRange.directive('guDateRange', [function () {
                 pikaEnd.show();
             });
 
-            $scope.$watch('pikaEndValue', function (pikaEndValue) {
-                var date = pikaEndValue && new Date(pikaEndValue);
-                pikaStart.setMaxDate(date || new Date());
```

Comment:
No this says that you can set a start date that is after today. When searching that concept didn't really make sense. 

Do you think it's a problem to have this spill over into the search functionality? I tried splitting that behavior out but found it was a little heavy for something that's effectively harmless